src/overlays/actors/ovl_Obj_Tokei_Step/z_obj_tokei_step.c

```diff
@@ -201,7 +201,7 @@ void ObjTokeiStep_Init(Actor* thisx, GlobalContext* globalCtx) {
         BgCheck3_LoadMesh(globalCtx, &this->dyna, &D_06000968);
         ObjTokeiStep_InitSteps(this);
         ObjTokeiStep_SetupBeginOpen(this);
-    } else if (!((((s32)gSaveContext.perm.day % 5) != 3) || (gSaveContext.perm.time >= 0x4000)) ||
+    } else if (!((CURRENT_DAY != 3) || (gSaveContext.perm.time >= 0x4000)) ||
                (s32)gSaveContext.perm.day >= 4) {
```

Comment:
Should be able to remove this cast as well.

Reply:
Good catch! Thanks